main/ZgatewayRF.ino

```diff
@@ -56,9 +56,11 @@ void RFtoMQTTdiscovery(SIGNAL_SIZE_UL_ULL MQTTvalue) { //on the fly switch creat
 #  endif
 
 void setupRF() {
-  //RF init parameters
+//RF init parameters
+#  ifndef ARDUINO_AVR_UNO // Space issues with the UNO
```

Comment:
Could you elaborate on this one please, we don't want this logs due to memory issue on Arduino UNO isn't it?

Reply:
Correct, with the active receiver additions the memory footprint for the UNO was exceeded and needed to trim some non-functional items to stay within the footprint.

Reply:
Pilight and RTL 433 are not compatible with UNO, should we rather ifndef the added code instead?

Reply:
Added that back and ifndef out other portions of the switching logic.  To keep the active receiver logic simple and avoid complex compiler definition logic a basic version of it is included with all receivers (RF, RF2, PiLight and RTL_433), with the code base expanding as each receiver is added.